Dockerfile.template

```diff
@@ -89,7 +89,10 @@ RUN noInstallRecommends="" && \
 		vim \
 		wget \
 		zip && \
+	# get the semi-official latest-stable git instead of using the old(er) version from the ubuntu distro
 	add-apt-repository ppa:git-core/ppa && apt-get install -y git && \
+	# get the semi-official latest-stable git-lfs too
```

Comment:
...so that it'd be more obvious to a reader why we're not simply using the git-lfs from the Ubuntu core distro here too.